src/Integrations/Integrations/Laminas/LaminasIntegration.php

```diff
@@ -0,0 +1,395 @@
+
+                    if (!in_array($eventName, $events)) {
+                        return;  // In other words, skips 'loadModule' and 'loadModule.resolve', which are too noisy
+                    }
+
+                    $span->name = "laminas.event.$eventName";
+                    $span->service = \ddtrace_config_app_name('laminas');
+                    $span->meta[Tag::COMPONENT] = 'laminas';
+                }
+            ]
+        );
+
+        trace_method(
+            'Laminas\Mvc\Application',
+            'run',
+            [
+                'prehook' => function (SpanData $span) use ($rootSpan) {
+                    $service = \ddtrace_config_app_name('laminas');
+                    $rootSpan->name = 'laminas.request';
+                    $rootSpan->service = $service;
+                    $rootSpan->meta[Tag::SPAN_KIND] = 'server';
+                    $rootSpan->meta[Tag::COMPONENT] = LaminasIntegration::NAME;
+
+                    $span->name = 'laminas.application.run';
+                    $span->resource = 'laminas.application.run';
+                    $span->type = Type::WEB_SERVLET;
+                    $span->service = $service;
+                    $span->meta[Tag::COMPONENT] = 'laminas';
+                }
+            ]
+        );
+
+        // MvcEvent::EVENT_ROUTE
+        trace_method(
+            'Laminas\Router\RouteInterface',
+            'match',
+            function (SpanData $span, $args, $retval) use ($rootSpan) {
+                $span->name = 'laminas.route.match';
+                $span->resource = \get_class($this) . '@match';
+                $span->meta[Tag::COMPONENT] = 'laminas';
+
+                /** @var RequestInterface $request */
+                $request = $args[0];
+
+                $method = $request->getMethod();
+
+                /** @var RouteMatch $routeMatch */
+                $routeMatch = $retval;
+                $routeName = $routeMatch->getMatchedRouteName();
+                $action = $routeMatch->getParam('action');
+                $controller = $routeMatch->getParam('controller');
+
+                if (method_exists($controller, $action . 'Action')) {
+                    trace_method(
+                        $controller,
+                        $action . "Action",
+                        function (SpanData $span) use ($controller, $action) {
+                            $span->name = 'laminas.controller.action';
+                            $span->resource = "$controller@{$action}Action";
+                            $span->meta[Tag::COMPONENT] = 'laminas';
+                        }
+                    );
+                }
+
+                $rootSpan->resource = "$controller@$action $routeName";
+
+                $rootSpan->meta[Tag::HTTP_METHOD] = $method;
+                $rootSpan->meta[Tag::HTTP_URL] = Normalizer::urlSanitize($request->getUriString());
+                $rootSpan->meta['laminas.route.name'] = $routeName;
```

Comment:
Check the span attributes tag doc I shared with you, I suppose those tags should be `http.` not `laminas.`